link/src/main/java/com/stripe/android/link/ui/verification/VerificationViewState.kt

```diff
@@ -0,0 +1,15 @@
+package com.stripe.android.link.ui.verification
+
+import androidx.compose.runtime.Immutable
+import com.stripe.android.core.strings.ResolvableString
+
+@Immutable
+internal data class VerificationViewState(
```

Comment:
What do you think about not having the default values on this data class? The advantage of that is that it makes it more obvious what the values are in the places where it's actually being used (e.g. the VerificationViewModel and VerificationScreen).

Default values are nice for convenience for users, but I think less important internally, where being explicit about what values are being used + having that clarity might be more valuable